shared/settings/archive/index.tsx

```diff
@@ -102,7 +231,9 @@ const Archive = C.featureFlags.archive
       const clearCompleted = C.useArchiveState(s => s.dispatch.clearCompleted)
 
       const jobMap = C.useArchiveState(s => s.jobs)
+      const kbfsJobMap = C.useArchiveState(s => s.kbfsJobs)
       const jobs = React.useMemo(() => [...jobMap.keys()], [jobMap])
+      const kbfsJobs = React.useMemo(() => [...kbfsJobMap.keys()], [jobMap])
```

Comment:
i don't think you really need this to be useMemo, its not a dependency of anything, if kbfsJobMap changes then this does to so you're not skipping any rendering